app/_hub/kong-inc/correlation-id/index.md

```diff
@@ -130,4 +130,20 @@ form parameter      | description
 
 ### Can I see my correlation ids in my Kong logs?
 
-The correlation id doesn't show up in the Nginx access or error logs. However, you can use this plugin along with one of the Logging plugins, or store the id on your backend.
+You can see your correlation ID in the Nginx access or error logs if you edit your Nginx logging parameters.
+
+To edit your Nginx parameters, do the following:
+
+1. In `/usr/local/share/lua/5.1/kong/templates/nginx_kong.lua`, add to a `log_format` section:
```

Comment:
```suggestion
1. Locate [{{site.base_gateway}}'s template files](/gateway/latest/reference/configuration/#custom-nginx-templates) and make a copy of `nginx_kong.lua`. 
2. Add a `log_format` section:
```
We should provide better guidelines on how to find the templates, and how to work with them. But since we don't want to have too much duplicate info, I think linking off to the config doc is sufficient.